index.html

```diff
@@ -5,41 +5,242 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>portfolio</title>
+    <title>Portfolio</title>
+    <!-- font awesome -->
     <script src="https://kit.fontawesome.com/6b55e5dec8.js" crossorigin="anonymous"></script>
+    <!-- css link -->
     <link rel="stylesheet" href="styles.css">
 </head>
 
 <body>
-    <!-- this is the navigation header -->
+
+
+    <!-- Navigation header -->
     <header>
         <div class="navigation">
             <a href="">WELCOME</a>
             <nav class="toggle">
-           <button class="menu"><i class="fa-solid fa-bars"></i></button>
+                <button class="menu"><i class="fa-solid fa-bars"></i></button>
             </nav>
         </div>
     </header>
 
     <!-- this is the section part -->
 
-    <section class="first-section">
-
-        <div class="section2">
-            <h1>I'm Ekene, <br> glad to see <br> you!</h1>
-            <p class="first-paragraph">I'm a software developer! I can help you build a product , feature or website Look through 
-                some of my work and experience! If you like what you see and have a project you need coded,
-                don't hesitate to contact me.</p>
-            <p class="second-paragraph">LETS CONNECT</p>
-            <ul class="social-media">
-                <li> <a href="#"><img src="./images/github.jpg" alt="github"></a> </li>
-                <li> <a href="#"><img src="./images/medium.jpg" alt="Linkedin"></a> </li>
-                <li> <a href="#"><img src="./images/twitter.jpg" alt="medium"></a> </li>
-                <li> <a href="#"><img src="./images/Vector.jpg" alt="twitter"></a> </li>
-            </ul>
-        </div>
-    </section>
+    <div class="background-color-container">
+
+
+        <section class="first-section">
+
+            <div class="section2">
+                <h1>I'm Ekene, <br> glad to see <br> you!</h1>
+                <p class="first-paragraph">I'm a software developer! I can help you build a product , feature or website
+                    Look through
+                    some of my work and experience! If you like what you see and have a project you need coded,
+                    don't hesitate to contact me.</p>
+                <p class="second-paragraph">LETS CONNECT</p>
```

Comment:
Kindly increase the spacing between the first project and the hero section to match the Figma design
| Your design | Figma design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/24703171/154681847-c4080b88-0c7b-4e35-b88a-000b8fea58b3.png) | ![Template 2-Mobile-Main](https://user-images.githubusercontent.com/24703171/154682085-63a189ce-ef4f-4585-9a22-02e80f78527a.png) |

Reply:
Hello sir. I have solved all issues you asked me to from your review.
1. I have increased the spacing between the first project and the hero section to match the Figma design
2. I wrapped Languages, Skills, and Frameworks inside a ul tag and changed the div tags to be li tags for semantic purposes.
3. I  changed the heading of the about section from h1 to h2, h5.